### ISSUE TITLE
SQLite Connector return code when list empty

### ISSUE THREAD

**Issue by J-8** [J-8/ShoppingList_backend#29]
To use the new response handling of the app, it would be helpful if the SQLite would return

``` php
return json_encode(array('type' => API_SUCCESS_LIST_EMPTY));
```

**Comment by J-8** [comment]
And also use the new response types and way MySQL does in general :)

**Comment by jklmnn** [comment]
Does this refer to #28 ? Otherwise I'm not 100% sure what you mean.

**Comment by J-8** [comment]
Yes, sorry for being unclear.

**Comment by jklmnn** [comment]
No problem, I will implement this as soon as I find time.

**Comment by J-8** [comment]
Thanks!

**Comment by jklmnn** [comment]
I resolved that with 0a6e996029debc307e0a444342d9de0ed87b7274
